src/verify_ce_config.py

```diff
@@ -63,75 +66,85 @@ def main():
-        try:
-            config_val = htcondor.param[attr]
-        except KeyError:
-            debug("Missing %s configuration value. " % attr +
-                  "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
-            continue
+    jr_transforms = {key : value for key, value in htcondor.param.items() if key.startswith("JOB_ROUTER_ROUTE_")}
 
-        # store the ads (iterating through ClassAdStringIterator consumes them)
-        try:
-            parsed_jr_ads[attr] = list(classad.parseAds(config_val))
-        except ValueError:
-            # We shouldn't ever get here since classad.parseAds() only raises ValueError when it's given
-            # non-string/non-file output and htcondor.param shouldn't contain such values
-            debug("Failed to parse %s configuration value. " % attr +
-                  "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
-
-        # If JRD or JRE can't be parsed, the job router can't function
-        if not parsed_jr_ads[attr]:
-            debug("Could not read %s in the HTCondor-CE configuration. " % attr +
-                  "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
-
-        if attr == "JOB_ROUTER_ENTRIES":
-            # Warn about routes we can find in the config that don't result in valid ads
-            malformed_entry_names = find_malformed_entries(config_val)
-            if malformed_entry_names:
-                warn("Could not read JOB_ROUTER_ENTRIES in the HTCondor-CE configuration. " +
-                     "Failed to parse the following routes: %s"
-                     % ', '.join(malformed_entry_names))
-
-            # Warn about routes specified by JOB_ROUTER_ROUTE_NAMES that don't appear in the parsed JRE.
-            # The job router can function this way but it's likely a config error
-            route_order = htcondor.param.get('JOB_ROUTER_ROUTE_NAMES', '')
-            if route_order:
-                missing_route_def = set(route_order.replace(',',' ').split()).difference(set(parse_route_names(config_val)))
-                if missing_route_def:
-                    warn("The following are specified in JOB_ROUTER_ROUTE_NAMES "
-                         "but cannot be found in JOB_ROUTER_ENTRIES: %s"
-                         % ', '.join(missing_route_def))
-
-    # Find all eval_set_ attributes in the JOB_ROUTER_DEFAULTS
-    eval_set_defaults = set([x.lstrip('eval_') for x in parsed_jr_ads['JOB_ROUTER_DEFAULTS'][0].keys()
-                             if x.startswith('eval_set_')])
-
-    # Find all default_ attributes used in expressions in the JOB_ROUTER_DEFAULTS
-    default_attr = set([re.sub(r'.*(default_\w*).*', 'eval_set_\\1', str(x))
+    # If JOB_ROUTER_ROUTE_<name> rules are defined, verify that these transforms exist
+    if jr_transforms:
+        route_names = htcondor.param.get("JOB_ROUTER_ROUTE_NAMES", "")
+        for name in route_names.split(" "):
```

Comment:
```suggestion
        for name in re.split(r'[, \t]+', route_names):
```

`JOB_ROUTER_ROUTE_NAMES` can be comma separated